apisix/utils/log-util.lua

```diff
@@ -120,7 +122,9 @@ local function get_full_log(ngx, conf)
         consumer = consumer,
         client_ip = core.request.get_remote_client_ip(ngx.ctx.api_ctx),
         start_time = ngx.req.start_time() * 1000,
-        latency = (ngx_now() - ngx.req.start_time()) * 1000
+        latency = latency,
+        upstream_latency = upstream_latency,
```

Comment:
I think we should discuss it in the mail list first when adding a new field to the full log format.

Is it commonly required? Or it is just used in some companies?

Reply:
ok, I'll continue to activate the previous discussion.